wallet-new/src/Cardano/Wallet/Kernel/DB/AcidState.hs

```diff
@@ -217,8 +221,7 @@ createPrefiltered initUtxoAndAddrs applyP accs = do
                 , _checkpointUtxoBalance = InDb $ Spec.balance utxo'
                 , _checkpointExpected    = InDb Map.empty
                 , _checkpointPending     = Pending . InDb $ Map.empty
-                -- TODO(@uroboros/ryan) proper BlockMeta initialisation (as part of CBR-239: Support history tracking and queries)
-                , _checkpointBlockMeta   = BlockMeta . InDb $ Map.empty
+                , _checkpointBlockMeta   = mempty
```

Comment:
This could do with a comment, saying something like "Since this is the first checkpoint _before_ we have applied any blocks, the block metadata is empty."